app/models/state_file_base_intake.rb

```diff
@@ -72,4 +75,16 @@ def full_name
       [@first_name, @middle_initial, @last_name].map(&:presence).compact.join(' ')
     end
   end
+
+  def disqualifying_eligibility_answer
+    disqualifying_eligibility_answers.each do |col, value|
+      return col if self.public_send(col) == value
+    end
+
+    nil
+  end
```

Comment:
This method will return the symbol for the column that had a disqualifying answer (for example `:eligibility_out_of_state_income`). We can use the symbol to look up a relevant message and translation.